Fabric/gradle.properties

```diff
@@ -1,9 +1,9 @@
-fabricVersion=0.85.0+1.20.1
-fabricLoaderVersion=0.14.21
-fabricLanguageKotlinVersion=1.9.4+kotlin.1.8.21
+fabricVersion=0.92.2+1.20.1
+fabricLoaderVersion=0.15.11
+fabricLanguageKotlinVersion=1.10.9+kotlin.1.9.0
```

Comment:
Why are these being updated?

Reply:
fabric language kotlin was to match kotlin being updated, don't know for the other teo